docs/examples/accounts

```diff
@@ -41,6 +41,7 @@
 #    ;stunpass=STUN/TURN/ICE-password
 #    ;stunserver=stun:[user:pass]@host[:port]  # see RFC 3986: Use of the format "user:password" in the userinfo field is deprecated.
 #    ;tcpsrcport=6060
+#    ;uas_req={yes, no, tls}  # default: no
```

Comment:
the current behaviour is to accept all incoming.

Reply:
We would like to propose to change the current behavior to not accept incoming MESSAGE requests per default.

Our reasoning is that we see this as non-RFC compliant. According to [RFC 3428 section 11](https://www.rfc-editor.org/rfc/rfc3428#section-11):
```
   This implies that MESSAGE requests have a greater need
   for security than most other SIP requests.  In particular, UAs that
   support the MESSAGE request MUST implement end-to-end authentication,
   body integrity, and body confidentiality mechanisms.
```
Some form of authentication of SIP MESSAGEs is a **MUST**, hence we don't think it should be enabled per default. If it is enabled, the user is responsible for configuring BareSIP to authenticate the SIP MESSAGEs, e.g. by setting `uas_user`/`uas_pass` or by setting TLS as the only transport protocol and enabling TLS client verification.

Let me know if this is okay. If not, we will change the default to `yes`.